pay-api/migrations/versions/2024_05_30_0672573574f6_20719_interim_statement.py

```diff
@@ -0,0 +1,27 @@
+"""20719 Add interim statement flag
+
+Revision ID: 0672573574f6
+Revises: 4b3bd37727a5
+Create Date: 2024-05-30 08:26:03.121991
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '0672573574f6'
+down_revision = '4b3bd37727a5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table('statements', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('is_interim_statement', sa.Boolean(), nullable=False,
```

Comment:
should we update all of current statements' is_interim_statement' field?

Reply:
Yes! the server_default will handle this on the line below